runtime/bcverify/staticverify.c

```diff
@@ -257,10 +291,22 @@ checkBytecodeStructure (J9CfrClassFile * classfile, UDATA methodIndex, UDATA len
 			}
 			info = &(classfile->constantPool[index]);
 			tag = (UDATA) info->tag;
-			if (!((tag == CFR_CONSTANT_Double)
-				  || (tag == CFR_CONSTANT_Long))) {
-				errorType = J9NLS_CFR_ERR_BC_LDC_NOT_CONSTANT__ID;
-				/* Jazz 82615: Set the constant pool index to show up in the error message framework */
+
+			if (CFR_CONSTANT_Dynamic == tag) {
+				J9CfrConstantPoolInfo *constantDynamicSignature = &classfile->constantPool[classfile->constantPool[info->slot2].slot2];
+				if (((flags & BCT_MajorClassFileVersionMask) < BCT_Java11MajorVersionShifted)
+				&& (0 != (flags & BCT_MajorClassFileVersionMask))
```

Comment:
Better use J9_ARE_ANY_BITS_SET() for (0 != (flags & BCT_MajorClassFileVersionMask)

Reply:
fixed